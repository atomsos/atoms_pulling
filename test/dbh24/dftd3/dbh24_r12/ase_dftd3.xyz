4

H       0.003272537417065      0.032739304231751      0.070981919413412
H       0.411061110799943      0.630906981538120      0.427838050446106
S       0.001201695838316      0.001416074937934      5.084508568121410
H      -0.001746219722180     -0.460952931884806      4.189267653198845
