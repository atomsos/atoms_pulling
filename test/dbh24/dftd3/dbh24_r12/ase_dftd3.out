  _________________________________
                                   
 |         DFTD3 V3.1 Rev 1        |
 | S.Grimme, University Bonn       |
 |         October  2015           |
 |   see dftd3 -h for options      |
  _________________________________

 Please cite DFT-D3 work done with this code as:
 S. Grimme, J. Antony, S. Ehrlich and H. Krieg,
 J. Chem. Phys. 132 (2010), 154104
 If used with BJ-damping cite also
 S. Grimme, S. Ehrlich and L. Goerigk,
 J. Comput. Chem. 32 (2011), 1456-1465
 For DFT-D2 the reference is
 S. Grimme, J. Comput. Chem., 27 (2006), 1787-1799
 For DFT-D3M or DFT-D3M(BJ) the reference is
 D.G.A. Smith, L.A. Burns, K. Patkowski, and 
 C.D. Sherrill, J. Phys. Chem. Lett. 7 (2016) 2197-2203

  files read :     
 ase_dftd3.xyz
C6 coefficients used:
           2  C6 for element            1
Z=  1 CN= 0.912     C6(AA)=    3.03
Z=  1 CN= 0.000     C6(AA)=    7.59
           3  C6 for element           16
Z= 16 CN= 0.000     C6(AA)=  134.01
Z= 16 CN= 0.995     C6(AA)=  131.00
Z= 16 CN= 1.990     C6(AA)=  125.81

#               XYZ [au]               R0(AA) [Ang.]  CN       C6(AA)     C8(AA)   C10(AA) [au] 
   1   0.00618   0.06187   0.13414   h     1.091    0.714         3.5        42.3       626.1
   2   0.77679   1.19224   0.80850   h     1.091    0.714         3.5        42.3       626.1
   3   0.00227   0.00268   9.60833   s     1.655    1.000       130.9      6415.4    385013.1
   4  -0.00330  -0.87107   7.91657   h     1.091    1.000         3.1        37.4       553.2

molecular C6(AA) [au] =       286.87
 --------------------------------------------------
 !! SOME DISTANCES VERY SHORT. CHECK COORDINATES !!
 --------------------------------------------------

           DFT-D V3
 DF pbe                                               
 parameters
 s6       :    1.0000
 s8       :    0.7220
 rs6      :    1.2170
 rs18     :    1.0000
 alpha6   :   14.0000
 alpha8   :   16.0000
 k1-k3    :   16.0000    1.3333   -4.0000
 Cutoff   :   95.0000 a.u.
 CN-Cutoff:   40.0000 a.u.

 Edisp /kcal,au:    -0.0779 -0.00012414

 E6    /kcal :    -0.0651
 E8    /kcal :    -0.0128
 % E8        : 16.41

 doing analytical gradient O(N^2) ...

 |G|=   5.3066263376148437E-004
gdisp time     0.0
 Cartesian gradient written to file dftd3_gradient
 normal termination of dftd3
